src/state.ts

```diff
@@ -26,3 +26,21 @@ export const state: State = {
   theme: undefined,
   translation: undefined,
 };
+
+let state: State | null = null;
+
+export const getCopyOfDefaultState = (): State => structuredClone(defaultState);
```

Comment:
I wasn't 100% sure on the use of `structuredClone`.  It is technically supported by greenfield browsers, but not IE.  Is that going to be a problem?

Reply:
If we're concerned about that is there a reason to not just use `JSON.parse(JSON.stringify(defaultState))`?

Reply:
IE is deprecated (last updated in 2013) and used by fewer than 0.39% of the world... I feel pretty comfortable not supporting IE.

Reply:
I guess it is only available in Node 17 and above, which hopefully people are using, but I imagine there are some who are not on that so I'll switch to use JSON cloning.